parsing/lexer.mll

```diff
@@ -462,23 +516,34 @@ rule token = parse
   | "{" (lowercase* as delim) "|"
       { let s, loc = wrap_string_lexer (quoted_string delim) lexbuf in
         STRING (s, loc, Some delim) }
-  | "{%" (extattrident as id) "|"
+ | "{" (ident_ext as raw_name) '|'
+      { let delim = validate_delim lexbuf raw_name in
```

Comment:
This rule could also be squashed with ASCII one ?

Reply:
I agree that performance don't matter that much here.